FileIO/GMSHInterface.cpp

```diff
@@ -74,6 +74,44 @@ GMSHInterface::GMSHInterface(GeoLib::GEOObjects & geo_objs,
 	}
 }
 
+int GMSHInterface::writeGeoFile(GeoLib::GEOObjects &geo_objects, std::string const& file_name)
+{
+	std::vector<std::string> names;
+	geo_objects.getGeometryNames(names);
+
+	if (names.size() == 0)
+	{
+		ERR ("No geometry information available.");
+		return 1;
+	}
+
+	bool const multiple_geometries = (names.size() > 1) ? true : false;
```

Comment:
ich need that information later again after i've cleared names for the merge-vector